tdrs-backend/tdpservice/users/models.py

```diff
@@ -17,3 +17,13 @@ class User(AbstractUser):
     def __str__(self):
         """Return the username as the string representation of the object."""
         return self.username
+
+    @property
+    def is_admin(self):
+        """Check if the user is an admin."""
+        # TODO: Probably better to change this to check for a permission
```

Comment:
@carltonsmith Let's clean up TODO comments before QASP review / merge.

Reply:
Yes, that makes sense

Reply:
@alexsoble This has been updated

Reply:
👍